addons/udes_stock/views/stock_picking.xml

```diff
@@ -59,7 +59,7 @@
 
             <!-- Partner field filtered to only see suppliers -->
             <xpath expr="//field[@name='partner_id']" position="replace">
-                <field name="partner_id" domain="[('supplier', '=', True)]" options="{'no_create': True}"/>
```

Comment:
If you add a partner through quick create it will default to a customer and not a vendor, could we add the following context?

`context="{'default_supplier': True, 'default_customer': False}"`

Reply:
Done